app/services/merchant_service.rb

```diff
@@ -0,0 +1,30 @@
+
+class MerchantService
+  class << self
+    def connection
+      Faraday.new(url: 'http://localhost:3000/api/v1/')
+    end
+
+    def parse_json(response)
+      JSON.parse(response.body, symbolize_names: true)
+    end
+
+    def merchants 
+      response = connection.get('merchants')
+
+      data = parse_json(response)
```

Comment:
(see also lines 21 and 27)
You shouldn't need a local varaible (`data`), because it's never being called. The methods would still return the value of `parse_json(response)` without it.